internal/core/converter/tensorflow/tensorflow_test.go

```diff
@@ -512,57 +782,122 @@ func Test_tensorflow_GetValue(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           inputs: nil,
-		       },
-		       fields: fields {
-		           exportDir: "",
-		           tags: nil,
-		           feeds: nil,
-		           fetches: nil,
-		           operations: nil,
-		           sessionTarget: "",
-		           sessionConfig: nil,
-		           options: nil,
-		           graph: nil,
-		           session: nil,
-		           ndim: 0,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           inputs: nil,
-		           },
-		           fields: fields {
-		           exportDir: "",
-		           tags: nil,
-		           feeds: nil,
-		           fetches: nil,
-		           operations: nil,
-		           sessionTarget: "",
-		           sessionConfig: nil,
-		           options: nil,
-		           graph: nil,
-		           session: nil,
-		           ndim: 0,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
```

Comment:
Could you please delete unnessesary fields to execute this test case?
https://github.com/vdaas/vald/pull/378/files#diff-d8102a1e4dec76fc40edb250de4d500eR749